tests/test_django/urls.py

```
@@ -177,50 +232,67 @@ class BookCrudl(CrudlController[Book], DefaultFilter):
+                    "inventory_number": Infer,
+                },
             }
         ),
         list_schema=Schema[Library](
             fields={
                 "id": Infer,
                 "name": Infer,
                 "address": Infer,
+                # TODO(phuongfi91): Test also this case
+                #  https://github.com/NextGenContributions/django-ninja-crudl/issues/11
+                # "book_copies": Infer,
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)